web/groups/hlt_summary.json

```diff
@@ -237,7 +237,7 @@
   "HLT1PFTau|": "others",
   "HLT1Photon|": "others",
   "HLT2CaloJetCaloJet|": "others",
-  "HLT2L1P2GTCandL1P2GTCandDZ|": "HLT",
+  "HLT2L1P2GTCandL1P2GTCandDZ|": "others",
```

Comment:
Why?

Reply:
> Some small changes in other modules were introduced by running the makefile in the groups folder, are connected to either mislabeling or wrong alphabetical order, and I think they make sense.

I can change it if it doesn't make sense.

Reply:
no, it was for my education - actually I ignored a MakeFile existed. if a previously agreed conversion exists I am happy to follow it.